strawberryfields/ops.py

```diff
@@ -2351,8 +2375,14 @@ class Interferometer(Decomposition):
               placed after all interferometers, and all beamsplitters decomposed into
               pairs of symmetric beamsplitters and phase shifters
 
+            - ``rectangular_compact'`` - rectangular mesh, with two independant phase shifts
+              placed inside each MZI, extra phase shifts on edges and at the input and output.
+
             - ``'triangular'`` - triangular mesh
 
+            - ``'triangular_compact'`` - triangular mesh, with two independant phase shifts
+             placed inside each MZI.
```

Comment:
```suggestion
            - ``'triangular_compact'`` - triangular mesh, with two independant phase shifts
              placed inside each MZI.
```
(the indentation error here was leading to a failure in the documentation CI: https://github.com/XanaduAI/strawberryfields/pull/584/checks?check_run_id=2632502387#step:4:1212)